src/main/java/org/broadinstitute/hellbender/tools/reference/ReferenceSequenceTable.java

```diff
@@ -0,0 +1,354 @@
+     *      EXACT_MATCH,
+     *      DIFFER_IN_SEQUENCE_NAMES,
+     *      DIFFER_IN_SEQUENCE,
+     *      DIFFER_IN_SEQUENCES_PRESENT,
+     *      SUPERSET,
+     *      SUBSET
+     *
+     * @return list of ReferencePairs with updated status sets
+     */
+    public List<ReferencePair> analyzeTable(){
+        List<ReferencePair> refPairs = generateReferencePairs();
+
+        for(TableRow row : tableByMD5.values()) {
+            for(ReferencePair pair : refPairs) {
+                int ref1Index = pair.getRef1ColumnIndex();
+                int ref2Index = pair.getRef2ColumnIndex();
+                TableEntry[] entries = row.getEntries();
+
+                TableEntry ref1Value = entries[ref1Index];
+                TableEntry ref2Value = entries[ref2Index];
+                if (!ref1Value.getColumnValue().equals(ref2Value.getColumnValue())) {
+                    pair.removeStatus(ReferencePair.Status.EXACT_MATCH);
+                }
+                if(!ref1Value.getColumnValue().equals(ref2Value.getColumnValue()) &&
+                        !(ref1Value.getColumnValue().equals(MISSING_ENTRY) || ref2Value.getColumnValue().equals(MISSING_ENTRY))){
+                    pair.addStatus(ReferencePair.Status.DIFFER_IN_SEQUENCE_NAMES);
+                }
+            }
+        }
+        for(ReferencePair pair : refPairs){
+            boolean superset = false;
+            boolean subset = false;
+            for(String sequenceName : tableBySequenceName.keySet()){
+                int ref1Index = pair.getRef1ColumnIndex();
+                int ref2Index = pair.getRef2ColumnIndex();
+                Set<ReferenceSequenceTable.TableRow> rows = queryBySequenceName(sequenceName);
+                int sequenceNameFound = 0;
+
+                for(TableRow row : rows) {
+                    TableEntry[] entries = row.getEntries();
+                    TableEntry ref1Value = entries[ref1Index];
+                    TableEntry ref2Value = entries[ref2Index];
+
+                    if((ref1Value.getColumnValue().equals(sequenceName) ^ ref2Value.getColumnValue().equals(sequenceName))
+                            && (ref1Value.getColumnValue().equals(MISSING_ENTRY) ^ ref2Value.getColumnValue().equals(MISSING_ENTRY))){
+                        sequenceNameFound++;
+                    }
+                    if(ref1Value.getColumnValue().equals(MISSING_ENTRY) ^ ref2Value.getColumnValue().equals(MISSING_ENTRY)){
+                        if(ref1Value.getColumnValue().equals(MISSING_ENTRY)){
+                            subset = true;
+                        }
+                        else if(ref2Value.getColumnValue().equals(MISSING_ENTRY)){
+                            superset = true;
+                        }
+                    }
+                }
+
+                if(sequenceNameFound == 2){
+                    pair.addStatus(ReferencePair.Status.DIFFER_IN_SEQUENCE);
+                } else if(sequenceNameFound == 1){
```

Comment:
Add a comment explaining the logic here (eg., why 2 means DIFFER_IN_SEQUENCE and 1 means DIFFER_IN_SEQUENCES_PRESENT)